### DISCUSSION TITLE
Skirtor agn.fracAGN parameter

### DISCUSSION THREAD

**Discussion by ageorgakakis** [mboquien/cigale#10] [Q&A] [answered]
Dear Cigale team

I would like to use CIGALE2022 with the skirtor and xray modules to fit the spectra of X-ray selected AGN by providing the "intrinsic" (i.e. obscuration corrected) X-ray flux as input. It is not clear to me why one of the skirtor parameters is the fraction of AGN emission in the infrared relative to total (agn.fracAGN). Given an X-ray flux the CIGALE xray module determines the f_2500 flux density and hence a nromalisation for the skirtor SED in the UV-IR regime, would that be correct? This intrinsic spectrum can then be further attenuated by dust (affecting mostly UV/optical part).  Why is there a need for an additional normalization parameter?

many thanks in advance, 

Antonis

**Comment by Guang91** [comment, marked as answer]
Dear Antonis, 

That's a really good question. 

Let me first explain how frac_AGN works. CIGALE first constructs an (unnormalized) galaxy component, and then it uses frac_AGN to add an AGN component. Finally, the code determines the normalization factor for the entire AGN+galaxy model by chi^2 fitting to the data. 

For your proposed scheme (using X-ray flux to normalize AGN), there are some issues I can think of. 
1. Given the error bars may be small or large, how to determine the specific value of AGN normalization? For large error bars, many values might be consistent with the data. 
2. The observed X-ray flux may also have galaxy X-ray binary contribution, how to deal with that?
3. Even after constructing the AGN component with X-ray flux, how to determine the normalization of the galaxy component? 

Thanks,
Guang

**Comment by ageorgakakis** [comment]
Dear Guang, 

many thanks for the quick reply. Your explanation makes sense. I was interpreting the a_ox of the xray module as a prior to the fit rather than a tool for estimating the X-ray flux given the AGN/galaxy SEDs. 

regards, 

Antonis
